analyzer/codechecker_analyzer/analysis_manager.py

```diff
@@ -565,27 +577,26 @@ def __create_timeout(analyzer_process):
             failure_type = "_crash"
 
         zip_file = result_base + zip_suffix + failure_type + '.zip'
-        zip_file = os.path.join(failed_dir, zip_file)
-
-        ctu_zip_file = result_base + ctu_suffix + failure_type + '.zip'
-        ctu_zip_file = os.path.join(failed_dir, ctu_zip_file)
+        failed_zip_file = os.path.join(failed_dir, zip_file)
```

Comment:
When there is no failure when analysing my project and I use the `--generate-reproducer` option it will create zip files inside my report directory, but it will contain the word `unknown`: `main.cpp_clangsa_cf726ed90650ca7a768d4364b14ccd3c.plist_unknown.zip`.  The reason is that the `failure_type` default value is `_unknown` even if the analysis was successfull. In case of failue maybe we can set it to an empty string.